### DISCUSSION TITLE
Should Address Labels for a Given Category be Additive or Mutually Exclusive?

### DISCUSSION THREAD

**Discussion by MSilb7** [duneanalytics/spellbook#2933] [Sector: Labels]
gm friends. @chuxinh and I were brainstorming some label ideas that we can sue to easily segment & get a better understanding of the frequent attributes of the users of a thing. (i.e. tell me the most common attributes of Optimism Quest participants).

For the class of labels that deal with "magnitude" (i.e. address age, transaction volume), what are people's thoughts on how to design them best?

Two options I see:
1. Make them "additive": For transaction age, have labels for '1+ Days old,' '1+ Months old,' '1+ Years old,' etc.
2. Make them "mutually exclusive": For transaction age, have labels '1 Day - 1 Month Old,' 1 - 2 years old,' etc.

In my opinion a benefit of 1 is that they may be easier to pump in to out-of-the-box algorithms/models (i.e. sklearn), but a downside is that they may create a bunch of clutter among labels. A pro of 2 is that there is less clutter, however it's more dependent on what ranges we set, and we may unintentionally over/under measure the long tail.

Open to thoughts here.

**Comment by bearishbuilder** [comment]
Why not do both versions? Put the mutually exclusive into one version, and the additive into the second version.
